apps/builder/src/locales/pt.ts

```diff
@@ -21,4 +25,142 @@ export default defineLocale({
+  'account.apiTokens.description':
+    'Esses tokens permitem que outros aplicativos controlem toda a sua conta e typebots. Tenha cuidado!',
+  'account.apiTokens.createButton.label': 'Criar',
+  'account.apiTokens.deleteButton.label': 'Excluir',
+  'account.apiTokens.table.nameHeader': 'Nome',
+  'account.apiTokens.table.createdHeader': 'Criado',
+  'account.apiTokens.deleteConfirmationMessage':
+    'O token {tokenName} será revogado permanentemente. Tem certeza de que deseja continuar?',
+  'account.apiTokens.createModal.createHeading': 'Criar Token',
+  'account.apiTokens.createModal.createdHeading': 'Token Criado',
+  'account.apiTokens.createModal.nameInput.label':
+    'Insira um nome único para o seu token para diferenciá-lo de outros tokens.',
+  'account.apiTokens.createModal.nameInput.placeholder':
+    'Ex. Zapier, Github, Make.com',
+  'account.apiTokens.createModal.createButton.label': 'Criar token',
+  'account.apiTokens.createModal.doneButton.label': 'Concluído',
+  'account.apiTokens.createModal.copyInstruction':
+    'Por favor, copie seu token e guarde-o em um lugar seguro.',
+  'account.apiTokens.createModal.securityWarning':
+    'Por motivos de segurança, não podemos mostrá-lo novamente.',
+  'account.preferences.graphNavigation.heading': 'Navegação do Editor',
+  'account.preferences.graphNavigation.mouse.label': 'Mouse',
+  'account.preferences.graphNavigation.mouse.description':
+    'Mova arrastando o quadro e amplie/reduza usando a roda de rolagem',
+  'account.preferences.graphNavigation.trackpad.label': 'Trackpad',
+  'account.preferences.graphNavigation.trackpad.description':
+    'Mova o quadro usando 2 dedos e amplie/reduza fazendo pinça',
+  'account.preferences.appearance.heading': 'Aparência',
+  'account.preferences.appearance.systemLabel': 'Sistema',
+  'account.preferences.appearance.lightLabel': 'Claro',
+  'account.preferences.appearance.darkLabel': 'Escuro',
+  'account.myAccount.changePhotoButton.label': 'Alterar foto',
+  'account.myAccount.changePhotoButton.specification':
+    '.jpg ou.png, máximo 1MB',
+  'account.myAccount.emailInput.disabledTooltip':
+    'A atualização do e-mail não está disponível. Entre em contato com o suporte se quiser alterá-lo.',
+  'account.myAccount.emailInput.label': 'Endereço de e-mail:',
+  'account.myAccount.nameInput.label': 'Nome:',
+  'analytics.viewsLabel': 'Visualizações',
+  'analytics.startsLabel': 'Inícios',
+  'analytics.completionRateLabel': 'Taxa de conclusão',
+  'auth.signin.heading': 'Entrar',
+  'auth.signin.noAccountLabel.preLink': 'Não tem uma conta?',
+  'auth.signin.noAccountLabel.link': 'Registe-se gratuitamente',
+  'auth.register.heading': 'Criar uma conta',
+  'auth.register.alreadyHaveAccountLabel.preLink': 'Já tem uma conta?',
+  'auth.register.alreadyHaveAccountLabel.link': 'Entrar',
+  'auth.error.default': 'Tente entrar com uma conta diferente.',
+  'auth.error.email':
+    'E-mail não encontrado. Tente entrar com um fornecedor diferente.',
```

Comment:
If this line makes reference to different auth providers, the word `fornecedor` must be changed to `provedor`.